manifests/plugin/tasks.pp

```diff
@@ -20,4 +23,15 @@
     enable => true,
     name   => $service,
   }
+  $cron_state = if $automatic_cleanup {
+    present
+  } else {
+    absent
+  }
+  file { 'foreman-tasks-cleanup-cron':
+    ensure  => $cron_state,
+    owner   => 'root',
+    path    => '/etc/cron.d/foreman-tasks-cleanup',
```

Comment:
Would ```foreman-tasks``` be more future proof? There could be another cronjob. We also often put the path in the file resource: ```file { '/etc/cron.d/foreman-tasks': }```